README.md

```diff
@@ -31,23 +28,34 @@ Pre-Requisites :
 * m2e 1.1 or later
 * m2eclipse-tycho 0.6 or later
 
-How to use
+Accessing the archetype from Eclipse
 -------------------
-
 In Eclipse, first add the Open Archetypes catalog :
 
-* On the Archetypes Preferences page (Window > Preferences > Maven > Archetypes), click on the "Add Remote Catalog..." button
+1. On the Archetypes Preferences page (Windows: `Window` > `Preferences`; OS X: `⌘,`), open `Maven` > `Archetypes`, click on the `Add Remote Catalog...` button
 
     - Catalog file : http://open-archetypes.github.com/maven-repo/releases/
-    - Description : Open Archetypes
+    - Description : "Open Archetypes"
```

Comment:
That might be confusing as you don't need to type the " in the wizard. 
If formatting is what you're after, then use `Open Archetypes`